DifferentLocation.java

```diff
@@ -0,0 +1,120 @@
+package location;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class DifferentLocation {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		WebDriver driver = new FirefoxDriver();
+
+		String appUrl = "http://www.tripfactory.com/";
+		driver.get(appUrl);
+
+		// Click on Location
+
+		driver.findElement(By.linkText("Location")).click();
+
+		List<WebElement> allElements = driver.findElements(By.xpath("//div[1]/div[3]/div/nav/ul/li[5]/ul"));
+
+		for (WebElement element : allElements) {
+			System.out.println(element.getText());
+		}
+
+		// Select the country
+
+		driver.findElement(By.partialLinkText("Australia")).click();

```

Comment:
You need to take benefit of the loop. There was no point in creating the List and the loop if you have to search again by "Australia"